.travis.yml

```diff
@@ -17,8 +17,8 @@ before_install:
 
 install:
 - make up
-- docker exec mgmt make -C /srv/$WP_ENV/jahia2wp bootstrap-mgmt
+- docker exec $(docker ps -q --filter "label=com.docker.compose.service=mgmt") make -C /srv/$WP_ENV/jahia2wp bootstrap-mgmt
```

Comment:
I don't find a clear documentation on this. 

It is mentionned in the [official doc](https://docs.docker.com/compose/install/#upgrading), and on [github](https://github.com/docker/compose/pull/1356)

Do you have anything better ? It would be nice to have a comment with a link here for the next readers.

Reply:
Indeed I had a surprisingly hard time finding documentation on the  `com.docker.compose.*` labels. I put what I found, including your second suggestion; kindly review again.